integration-tests/src/server.js

```diff
@@ -20,26 +21,44 @@ const app = express()
 const server = http.createServer(app)
 server.setTimeout(serverTimeout)
 
+const firestore = new Firestore()
+
 const testTimeout = async timeout => {
   await delay(timeout)
   throw new Error(`Test timeout after ${timeout}ms`)
 }
 
+const reportTest = (testGroup, startTime, host, testName, errorMessage) => {
+  const endTime = new Date()
+  const testReport = { testGroup, runtimeInMilliseconds: endTime - startTime, status: errorMessage ? 'error' : 'success', testName, errorMessage, host }
+  firestore.collection('tests').doc().create(testReport)
```

Comment:
This call is async, so we should make sure it gets properly `await`-ed all the way up to the original caller.

Reply:
I was wondering about this one, because I could see a world where we didn't want to await on firestore (just fire and forget)

Reply:
In this case, the expectation is that the DB write will succeed, and it should be very fast, so waiting gives us a potential way to catch errors sooner in case they happen.